src/plugin-slots/PageBannerSlot/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode } from 'react';
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+import { PageBanner } from '@openedx/paragon';
+
+export interface PageBannerSlotProps {
+  show: boolean;
+  dismissible: boolean;
```

Comment:
I don't think `dismissable` should be a prop for the plugin slot. First of all, it's never false, and second of all, it should be up to the plugin contents to decide whether to show a dismiss button or not.